resources/chefdk/msi/localization-en-us.wxl.erb

```diff
@@ -20,9 +20,12 @@
   <String Id="FeaturePSModuleName"><%= friendly_name %> PowerShell wrappers</String>
   <String Id="FeatureChefDkEnvHacks">ChefDK Environment Customizations</String>
   <String Id="FeatureChefDkEnvHacksDesc">Allows ChefDK to try to find a suitable HOME drive when running Ruby programs.</String>
-  
-  <String Id="ChefDkStartMenuShortcutDefName"><%= friendly_name %></String>
-  <String Id="ChefDkStartMenuShortcutDefDescription"><%= friendly_name %> Command Prompt</String>
+
+  <String Id="ChefDkShortcutDefName"><%= friendly_name %></String>
+  <String Id="ChefDkShortcutDefDescription"><%= friendly_name %> Command Prompt</String>
+
   <String Id="FeatureChefDkStartMenuShortcut">Start Menu Shortcut</String>
-  <String Id="FeatureChefDkStartMenuShortcutDescription">Install a shortcut on the start menu</String>
+  <String Id="FeatureChefDkStartMenuShortcutDescription">Install a ChefDK shortcut on the user's start menu. The shortcut will launch powershell (or conemu, if installed) with the ChefDK in the PATH and ChefDK powershell extensions in it.</String>
```

Comment:
change capitalization: "PowerShell" and "ConEmu"

I actually like the original simpler wording.. "Install a shortcut on the Start Menu".